tox.ini

```diff
@@ -11,6 +11,7 @@
 #
 
 [tox]
+minversion = 3.4.0
```

Comment:
minversion should be 3.5.0 as we use type hints and annotations which do not exists on py3.4

Reply:
Oh right. Good catch! I've corrected the line now.